palm/plugins/dbt/commands/cmd_cleanup.py

```diff
@@ -9,7 +9,7 @@
 def cli(ctx):
     """Removes any artifacts from Snowflake related to the current branch."""
 
-    cmd = "dbt clean && dbt deps && dbt run-operation drop_branch_schemas"
+    cmd = "dbt clean && dbt run-operation drop_branch_schemas"
```

Comment:
oof on this one... if dbt clean is cleaning up deps then potentially run-operation, if it relies on imported packages' macros, might fail

Reply:
maybe we drop branch schemas _then_ cleanup